slackbridge/bots.py

```diff
@@ -149,6 +151,30 @@ def nickserv_auth(self):
                 self.target_group_nick,
                 self.nickserv_password,
             ))
+    def privmsg(self, user, channel, message):
```

Comment:
Please add an empty line above this method so that it's clear that it is separate from the previous method.

Reply:
fixed